src/main/java/com/treasure/hunt/view/MainController.java

```diff
@@ -399,6 +421,14 @@ public void onLoadGame(ActionEvent actionEvent) {
         FileService.getInstance().loadGameManager();
     }
 
+    public void openPreferences() {
+        try {
+            SettingsWindow.show();
```

Comment:
I made an event that you can throw with opens the window in the main window. Maybe that is nicer. Dont need to do it